docs/api/Model.md

```diff
@@ -1,26 +1,21 @@
 # Model
 
-The Model object represents one DynamoDB table. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
+The Model object represents a table in DynamoDB. It takes in both a name and a schema and has methods to retrieve, and save items in the database.
 
 ## new dynamoose.Model(name, schema[, config])
 
-This method is the basic entry point for creating a model in Dynamoose. When you call this method a new model is created, and it returns a Document initializer that you can use to create instances of the given model.
+This method is the basic entry point for creating a model in Dynamoose. When you call this method a new model is created, and it returns an initializer that you can use to create instances of the given model.
 
-The `schema` parameter can either be an object OR a Schema instance. If you pass in an object for the `schema` parameter it will create a Schema instance for you automatically.
+The `schema` parameter can either be an object OR a Schema instance, see [Schema](Schema.md) for details. If you pass in an object for the `schema` parameter it will create a Schema instance for you automatically.
 
 ```js
-const dynamoose = require("dynamoose");
-
 const Cat = new dynamoose.Model("Cat", {"name": String});
-```
-
-```js
-const dynamoose = require("dynamoose");
-
+const Cat = new dynamoose.Model("Cat", {"name": String}, { create: false });
```

Comment:
Any reason for adding this? Right below we go over the `config` parameter. Why introduce it here? And why just `create`?

```suggestion
const Cat = new dynamoose.Model("Cat", {"name": String}, {"create": false});
```

Reply:
I think it was just to show that a Model accepts options. Can be removed if you think it is not necessary. I know the function definition is in the title, but sometimes one overreads it in the title and then sees the example and thinks it has no config options.

Reply:
You used `create: false` in a lot of examples so I did that, too for consistency.